### PR TITLE
feat(github/issuetemplates: Update bug report templates to include simulator versions

### PR DESCRIPTION
**Pull request by CaptCrystal**
Fixes #[issue_no]

## Summary of Changes

- Add the prompt into bug_report.yaml

## Screenshots (if necessary)
image

## References
Not required

## Additional context

Discord username (if different from GitHub):

## Testing instructions
N/A (You can't test this)

## How to download the PR for Testing

Every new commit to this PR will cause a new A330-900neo artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **HEADWIND-A339** download link at the bottom of the page

### CODE REVIEW

**Comment by Revyn112** [comment]
@CaptCrystal Nice one! Can you also add a dropdown for aircraft type selection between A339X and SU95X?
As we use this as monorepo for every aircraft now :)

**Comment by CaptCrystal** [comment]
> @CaptCrystal Nice one! Can you also add a dropdown for aircraft type selection between A339X and SU95X? As we use this as monorepo for every aircraft now :)

I can!

**Comment by CaptCrystal** [comment]
image

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -4,12 +4,32 @@ labels: [Bug]
...
+    required: true
 - type: dropdown
   id: version
   attributes:
     label: Aircraft Version
-    description: What version of the A330-900 are you using?
+    description: What version of the aircraft are you using?
     options:
     - Stable
     - Development
```
Comment (Revyn112) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:35]: This should be renamed to "Pre-Release" as we not providubg development version anymore.

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -4,12 +4,32 @@ labels: [Bug]
...
+  validations:
+    required: true
+- type: dropdown
+  id: aircraft
+  attributes:
+    label: Aircraft
+    description: What aircraft are you using?
+    options:
+    - A330-900x
+    - SU95x
```
Comment (Revyn112) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:25]: Should be named "SU95X"

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 3]
```diff
@@ -4,12 +4,32 @@ labels: [Bug]
...
+    - 2024
+  validations:
+    required: true
+- type: dropdown
+  id: aircraft
+  attributes:
+    label: Aircraft
+    description: What aircraft are you using?
+    options:
+    - A330-900x
```
Comment (Revyn112) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:24]: Should be named "A339X"

**Review by Revyn112 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 3 above)
Comment (CaptCrystal) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:24]: Will fix that now!

**Review by Revyn112 — APPROVED** [review]

**Review by Revyn112 — APPROVED** [review]

**Pull request merged**